Generation/src/components/PythiaInterface.cpp

```diff
@@ -39,7 +39,6 @@ PythiaInterface::PythiaInterface(const std::string& type, const std::string& nam
 
   declareProperty("VertexSmearingTool", m_vertexSmearingTool);
   declarePrivateTool(m_vertexSmearingTool, "FlatSmearVertex/VertexSmearingTool");
-  declareProperty("hepmc", m_hepmchandle, "The HepMC event (output)");
```

Comment:
This seems to have slipped through during the restructuring of the Generation package. Good catch!